ui/packages/catalog-ui-search/src/main/webapp/react-component/location/gazetteer.js

```diff
@@ -197,7 +197,7 @@ class Gazetteer extends React.Component {
     })
   }
   isMultiPolygon(coordinates) {
-    return coordinates[0][0][0] !== null && coordinates[0][0][0][0] !== null
+    return coordinates[0][0][0] !== undefined && coordinates[0][0][0][0] !== undefined
```

Comment:
❓ Is this a case where we wouldn't need to support both `null` and `undefined`? I'm not sure how many different dependencies we have that deal with geometry and if they handle it differently.

Reply:
To my knowledge this code is only used for the online/offline gazetteer which both return results in the same format: a 3D array for polygons and a 4D array for multipolygons. I'm not sure whether that answers your question though.

Reply:
I think `undefined` should be sufficient 👍